server/controllers/stock/core.js

```diff
@@ -480,6 +479,59 @@ async function getMovements(depotUuid, params) {
   return movements;
 }
 
+/**
+ * @function listLostStock
+ *
+ * This function returns a list of stock lost between purchase and delivery
+ */
+function listLostStock(params) {
+  const sql = `
+    SELECT
+      BUID(dest.uuid) as uuid, dest.date, dest.description, dest.period_id,
+      dest.document_uuid, dest.depot_uuid, dest.unit_cost,
+      dd.text AS destDepot, od.text AS srcDepot,
+      dest.quantity AS quantityRecd, ex.quantity AS quantitySent,
+      IFNULL((ex.quantity - dest.quantity), 0) AS quantityDifference,
+      i.code AS inventory_code, i.text AS inventory_name,
+      l.label AS lotLabel, l.expiration_date AS expirationDate,
+      dest.unit_cost, (dest.quantity * dest.unit_cost) AS total_cost
+    FROM stock_movement dest
+    JOIN depot dd ON dd.uuid = dest.depot_uuid
+    JOIN depot od ON od.uuid = dest.entity_uuid
+    JOIN lot l ON l.uuid = dest.lot_uuid
+    JOIN inventory i ON i.uuid = l.inventory_uuid
+    JOIN (
+      SELECT m.document_uuid, m.lot_uuid, m.quantity, m.unit_cost
+      FROM stock_movement m
+      WHERE m.is_exit = 1 AND m.flux_id = ${flux.TO_OTHER_DEPOT}
+    ) ex ON ex.document_uuid = dest.document_uuid AND ex.lot_uuid = dest.lot_uuid
+  `;
+
+  db.convert(params, ['depot_uuid']);
+
+  const { depotRole } = params;
+  delete params.depotRole;
+
+  params.fake_field = true;
+  const filters = new FilterParser(params, { tableAlias : 'dest' });
+
+  filters.custom('fake_field', 'dest.flux_id = (?) AND IFNULL((ex.quantity - dest.quantity), 0) <> 0',
```

Comment:
You probably should add a comment describing what you are doing here.  I would also rename this from "fake_field" to `unbalanced_quantities` or something similar that is related to the thing being looked up.

Reply:
Done.